Makefile

```diff
@@ -0,0 +1,27 @@
+# Makefile
+
+# Define variables
+PYTHON = python
+SETUP = setup.py
+
+# Default target
+all: build
+
+# Build target
+build:
+	$(PYTHON) $(SETUP) build
+
+# Clean target to remove build artifacts
+clean:
+	$(PYTHON) $(SETUP) clean
+
+# Install target (optional, if you want to include it)
+install:
+	$(PYTHON) $(SETUP) install
+
+# Uninstall target (optional, if you want to include it)
+uninstall:
+	$(PYTHON) $(SETUP) uninstall
+
+.PHONY: all build clean install uninstall
+
```

Comment:
I'd prefer if we didn't include the Makefile as I don't think @bsteffensmeier or myself will ever test or maintain it to make sure it still works.  Also I believe with newer versions of setuptools we're no longer supposed to call `python setup.py target` and this is one more thing we'd have to maintain as we shift away from that.